gax/src/main/java/com/google/api/gax/rpc/OperationApi.java

```diff
@@ -30,36 +30,13 @@
 package com.google.api.gax.rpc;
 
 import com.google.api.core.BetaApi;
-import java.io.IOException;
-import java.util.concurrent.ScheduledExecutorService;
 
-/** The settings used to create a Transport. */
 @BetaApi
-public interface TransportProvider {
+public interface OperationApi {
```

Comment:
I will move operation stuff that is not rpc-specific (e.g. callable-related) to `com.google.api.gax.longrunning`.

Also, related to the `OperationApi` name, I will rename that to `LongRunningClient`. It does need to stay under `com.google.api.gax.rpc` though since it references `UnaryCallable`.